core/src/test/scala/unit/kafka/server/ApiVersionsRequestTest.scala

```diff
@@ -47,7 +47,7 @@ object ApiVersionsRequestTest {
     List(ClusterConfig.defaultBuilder()
       .setTypes(java.util.Collections.singleton(Type.ZK))
       .setServerProperties(serverProperties)
-      .setMetadataVersion(MetadataVersion.IBP_4_0_IV0)
+      .setMetadataVersion(MetadataVersion.IBP_3_9_IV0)
```

Comment:
Could we add some comments on which MV this test should set? Is it the first unstable MV? Could we add sth like MetadataVersion.earliestTesting() to avoid having to keep changing it?

Reply:
Good point. I'll just set `latestTesting` so that we don't have to keep updating it.